### PR TITLE
Add channels.counters, groups.counters, im.counters

### PR DESCRIPTION
**Pull request by xbolshe**
Docs for PR [#9679](https://github.com/RocketChat/Rocket.Chat/pull/9679)

### CODE REVIEW

File: developer-guides/rest-api/channels/counters/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+## Example Result
+
+```json
+{
+   "joined":true,
+   "members":59,
+   "unreads":0,
+   "msgs":236,
```
Comment (MarcosSpessatto) [inline developer-guides/rest-api/channels/counters/README.md]: Hi @xbolshe maybe you forgot `unreadsFrom` property in this response.

Comment (xbolshe) [reply developer-guides/rest-api/channels/counters/README.md]: Hi @MarcosSpessatto Thanks for a detection of issue. Now it is fixed as well as docs.

**Review by MarcosSpessatto — APPROVED** [review]
Thanks for contribution @xbolshe.

**Review by MarcosSpessatto — APPROVED** [review]

File: developer-guides/rest-api/channels/counters/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+msgs - count of messages in the room
+latest - end date-time of unread interval for specified user (or date-time of last posted message)
+userMentions - count of user mentions in messages
+```
+
+## Change Log
+
+| Version | Description |
+| :--- | :--- |
+| 0.62.0 | Added |
```
Comment (rafaelks) [inline developer-guides/rest-api/channels/counters/README.md]: I believe this one was added to 0.65.0?

File: developer-guides/rest-api/groups/counters/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+msgs - count of messages in the room
+latest - end date-time of unread interval for specified user (or date-time of last posted message)
+userMentions - count of user mentions in messages
+```
+
+## Change Log
+
+| Version | Description |
+| :--- | :--- |
+| 0.62.0 | Added |
```
Comment (rafaelks) [inline developer-guides/rest-api/groups/counters/README.md]: I believe this one was added to 0.65.0?

File: developer-guides/rest-api/im/counters/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+msgs - count of messages in the room
+latest - end date-time of unread interval for specified user (or date-time of last posted message)
+userMentions - count of user mentions in messages
+```
+
+## Change Log
+
+| Version | Description |
+| :--- | :--- |
+| 0.62.0 | Added |
```
Comment (rafaelks) [inline developer-guides/rest-api/im/counters/README.md]: I believe this one was added to 0.65.0?

File: developer-guides/rest-api/channels/counters/README.md (hunk 2 above)
Comment (xbolshe) [reply developer-guides/rest-api/channels/counters/README.md]: It is hard to predict a release version. Of course need to update a version to a correct value.

**Comment by xbolshe** [comment]
@rafaelks Versions are updated to 0.65.0.

**Pull request merged**